src/main/kotlin/com/msg/gauth/global/security/SecurityConfig.kt

```diff
@@ -49,6 +49,7 @@ class SecurityConfig(
             .antMatchers(HttpMethod.PATCH, "/auth/password/initialize").permitAll()
             .antMatchers(HttpMethod.PATCH, "/auth/image").permitAll()
             .antMatchers(HttpMethod.DELETE, "/auth/image").permitAll()
+            .antMatchers(HttpMethod.PATCH, "/auth/password").permitAll()
```

Comment:
```suggestion
            .antMatchers(HttpMethod.PATCH, "/auth/password").authenticated()
```
이거 authenticated로 해야할거같아요!